src/Aircraft/Main.java

```diff
@@ -0,0 +1,198 @@
+        // If the aircraft is a helicopter, add the set altitude command
+        if (aircraft instanceof Helicopter){
+            commands.add("Set altitude");
+        }
+
+        // When there is still distance left and user has not quit, run the commands menu
+        boolean isQuit = false;
+        while (!isQuit && !(aircraft.getDistance() == 0)){
+            System.out.println("Choose your command: ");
+            
+            // Print commands
+            for (int i = 0; i < commands.size(); i++){
+                System.out.print(i + ".)" +commands.get(i) + "   ");
+            }
+            System.out.println();
+            int intIndex = Integer.parseInt(keyboard.readLine());
+            // Cases for each of the commands
+            // For each case run a different method
+            switch (intIndex){
+                case 0: 
+                  addPassenger(aircraft, keyboard);
+                  break;
+                case 1:
+                  flight(aircraft, keyboard);
+                  break;
+                case 2:
+                  System.out.print(aircraft.status());
+                  break;
+                case 3:
+                  isQuit = true;
+                  System.out.println("Terminating flight...");
+                  break;
+                case 4:
+                  if (aircraft instanceof Helicopter){
+                    setAltitude((Helicopter)aircraft, keyboard);
+                  }
+                  break;                  
+            }
+        }
+    }
+
+
+    /**
+    * Asks the user for information about the passenger and adds that information into an aircraft
+    *
+    * @param aircraft - the aircraft
+    * @param keyboard - reads the user's input 
+    */	
+    public static void addPassenger(Aircraft aircraft, BufferedReader keyboard) throws IOException {
+        
+        // Asks for the passenger's name and age and records it
+        System.out.print("What is the passenger's name? ");
+        String strName = keyboard.readLine();
+        System.out.print("What is the passenger's age? ");
+        int intAge = Integer.parseInt(keyboard.readLine());
+        int intIndex = -1;        
+
+        // Do the following as long as intIndex == -1 or intIndex >= jobsList.size()
+        do{
+            System.out.println("What is the passenger's job? ");
+            // Print out a menu with the job options the user can use
+            for (int i = 0; i < jobsList.size(); i++){
+                System.out.print(i + ".)" +jobsList.get(i).getName()+ "   ");
+            }
+            System.out.println();
+            intIndex = Integer.parseInt(keyboard.readLine());
+        } while (intIndex == -1 || intIndex >= jobsList.size());
+        
+        // Create a job object, set it equal to the user option, and add it to a new passenger 
+        Job job = jobsList.get(intIndex);
+        Passenger passenger = new Passenger(strName, intAge, job);
```

Comment:
this method can be further encapsulated and its purpose (get passenger details) can be part of the addPassenger method in the Aircraft class.  There is no need for all these details to be retrieved in the main class.